frontend/lib/resolution/Resolver.cpp

```diff
@@ -1410,13 +1385,14 @@ void Resolver::resolveTypeQueries(const AstNode* formalTypeExpr,
 
       // Now, consider the formals
       int nActuals = call->numActuals();
+      int faActualIndex = 0; // faMap doesn't count '?' for actual indexing.
```

Comment:
I wonder if we should change this (in a separate effort). I'm guessing this would have been motivated by the use of ``?`` in type constructors, rather than function calls.